docs/explanation/stable-release-updates.rst

```diff
@@ -1,2 +1,257 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To investigate why phasing stopped, use the phased updates report.
+
+When investigating an increased rate of crashes, focus on the crashes with the highest
+number of occurrences. Examine the occurrences table to determine if these crashes are
+happening more frequently with the updated version of the package. If they are,
+investigate the cause and address the crash in a follow-up :term:`SRU`. If not,
+`contact the Ubuntu SRU Team`_ about overriding the crash report.
+
+For new errors, verify that they're indeed new by reviewing the versions table and checking
+the ``Traceback`` or ``Stacktrace`` to determine if the error originates from the updated
+package or an underlying library. If you believe the error wasn't caused by the update,
+you can  `contact the Ubuntu SRU Team`_ to override the crash.
+
+Overriding halted phased updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Overriding halted phasing is similar to handling :term:`autopkgtest` failures.The phased
+update machinery uses a file named ``phased-update-overrides.txt``, a simple CSV file
+containing lines of the form ``source package``, ``version``, and ``$THING_TO_IGNORE``.
+
+``$THING_TO_IGNORE`` can either be an ``errors.ubuntu.com`` problem URL to ignore or
+``increased-rate``.
+
+.. _Verification:
+
+Verification
+------------
+
+:term:`SRU` verification should be done in a software environment that closely resembles
+that which will exist after the package is copied to
+:ref:`updates <ArchivePockets_Updates>`.
+Generally, this will be with a system that's up to date from
+:ref:`release <ArchivePockets_Release>`,
+:ref:`security <ArchivePockets_Security>`, and
+:ref:`updates <ArchivePockets_Updates>`.
+It shouldn't include other packages from
+:ref:`proposed <ArchivePockets_Proposed>` or
+:ref:`backports <ArchivePockets_Backports>`,
+with one exception: other packages built from the affected source package must
+be updated if they're generally installed.
+
+If the fix is sufficient, the :term:`SRU Verification Team` will update the bug status
+to ``In Progress``, and change the  ``verification-needed-$RELEASE`` tag to
+``verification-failed-$RELEASE``. If the fix is sufficent, the :term:`SRU Verification Team`
+will tag it as ``verification-done-$RELEASE``.
+
+If you encounter a regression in a package uploaded to
+:ref:`proposed <ArchivePockets_Proposed>`, do the following:
+
+1. File a bug report describing the nature of the regression.
+#. Tag the bug as ``regression-proposed``.
+#. `Ask a bug supervisor`_ to target the bug to the appropriate :ref:`Ubuntu releases <UbuntuReleases>`.
+#. Follow up on the :term:`SRU` bug report referenced from the package changelog, pointing
+   to the new bug. If there is more than one bug in the :term:`SRU` changelog, follow up to
+   the bug that is most closely related to the regression.
+#. Set the ``verification-failed-$RELEASE`` tag on the corresponding :term:`SRU` bug report.
```

Comment:
nitpick: This page already established the acronym. There is no value in linking to the glossary.

```suggestion
#. Set the ``verification-failed-$RELEASE`` tag on the corresponding SRU bug report.
```